.github/workflows/aks-letsencrypt.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           go-version: ${{ env.SETUP_GO_VERSION }}
 
       - name: Setup Ginkgo Test Framework
-        run: go install github.com/onsi/ginkgo/v2/ginkgo@v2.8.0
+        run: go install github.com/onsi/ginkgo/v2/ginkgo@v2.9.5
```

Comment:
Why not bump to 2.10.0?

Reply:
Ah, actually I think I choose the `2.9.5` because it was the one that was prompted to upload from some logs, and I thought it was the latest one. If you're ok with that I would postpone the ginkgo update in another PR just because I hope that tests are not going to fail. :smile: